virtwho/log.py

```diff
@@ -282,6 +284,8 @@ def get_file_handler(cls, name, config=None):
     @classmethod
     def get_stream_handler(cls, name):
         if cls._stream_handler is not None:
+            if cls._debug and cls._status:
```

Comment:
Why is stream set to `stderr` here, when the `cls._stream_handler` already exists? This block code is used for caching.